CSLI PDD

jprks opened the pull request:
### Authors
tag authors here, including their github username 
* James Parkus @jprks
* Austin Bodzas @Beaudidly 

### Abstract
The purpose of this project is to go through the design stages of the CSLI\@. The team will choose a scientific payload, engineer the CubeSat to house and function the payload, begin writing the CSLI by excluding the funding section, and perform a full cost estimation. This stage will leave off with a firm idea of the road ahead for funding and fabrication. The next stage will start with funding based off the cost estimations this team makes.

Commit:
Added CSLI PDD. Initial project commit.

Commit:
Minor edit

venku122 commented on CSLI/sample-formats.bib:
```diff
@@ -0,0 +1,155 @@
+@article{article,
```
This can be removed

runphilrun replied:
delete this file

venku122 commented on CSLI/sample-with-examples.bib:
```diff
@@ -0,0 +1,195 @@
+@manual{lamp87,
```
This can be removed.

runphilrun replied:
delete this file

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+\hyphenation{explor-ation}
+
+\begin{abstract}
+The purpose of this project is to go through the design stages of the CSLI\@. The team will choose a scientific payload, engineer the CubeSat to house and function
```
What does 'engineer' mean in this context. Overall, I am a bit confused reading this PDD. Is this a[ Pre-Phase A/Phase A](https://nodis3.gsfc.nasa.gov/policy_letters/NM_7120-81_.pdf) research project, or is actual system and subsystem design occurring? CAD and electrical engineering skills are referenced later in the document which leads me to believe the latter. Is that correct?

runphilrun replied:
CSLI is usually a means to an end, like how USIP was an avenue to do CSTARS. Is this paper outlining why SPEX should design _something_ for CSLI or outlining a specific project (or family of projects) that make use of CSLI to meet their goals?

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+
+\begin{abstract}
+The purpose of this project is to go through the design stages of the CSLI\@. The team will choose a scientific payload, engineer the CubeSat to house and function
+the payload, begin writing the CSLI by excluding the funding section, and perform a full cost estimation. This stage will leave off with a firm idea of the road ahead for funding and fabrication. The next stage will start with
```
Again, a bit of confusion here, as 'writing the CSLI' I believe references the grant proposal document, while other sections of this PDD focus on engineering design and development. What exactly will be done at the end of this project? High-level CAD models, sub-assemblies, PCB designs?

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+\label{sec:introduction}
+  % The introduction is a place to give background and context before diving into the subject matter.
+  % Establish context for the work you are about to propose and the main ideas of the proposition itself.
+\IEEEPARstart{T}{he} CubeSat Launch Initiative will be a great opportunity for RIT SPEX members to use their technical skills to make an electromechanical system.
```
Some more context on the CSLI proposal process including specific details and timelines would be appreciated. The text relies on a lot of extra context to be properly understood. Imagine showing this to an RIT professor, who had never heard of the CSLI program, but might be interested in flying a payload or mentoring the team. What would you need to tell them to get them on the same page?

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+\IEEEPARstart{T}{he} CubeSat Launch Initiative will be a great opportunity for RIT SPEX members to use their technical skills to make an electromechanical system.
+CubeSats were designed with the intent of creating a low-cost opportunity for launching satellites, geared toward allowing university students to put there skills to
+real use. The CSLI gives the team a free ride to space, thus eliminating a massive amount of cost to put a satellite in orbit. RIT SPEX will create a team
+to design, fabricate, and launch RIT's first CubeSat. There are several phases to this mission and will not be restricted to a tight-timeline. The team will be given ample
```
As I mentioned before, references to 'engineer' 'design' 'fabricate' and 'launch' are used throughout. What specific activities are planned for this project? Consider moving 'CSLI Part 2-N' activities to the long-term vision section to indicate they are follow-on projects.

runphilrun replied:
"this mission" hasn't been described yet beyond a cubesat following the CSLI rules/SOP. can you describe some goals for the scientific objective?

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+  % Sometimes our expectations are wrong and that can be just as successful as getting data we thought we'd see.
+  % What matters are what questions you intend to answer.
+  % This is the main purpose or main goal the project hopes to achieve.
+This part of the project, Phase I, will concern itself with the design and research of the CubeSat. This spans everything from picking/researching a payload
```
How will payloads be researched? What activities will be performed to learn more about payload ideas? What does the selection process look like?

runphilrun replied:
picking/researching --> choosing

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+  % What matters are what questions you intend to answer.
+  % This is the main purpose or main goal the project hopes to achieve.
+This part of the project, Phase I, will concern itself with the design and research of the CubeSat. This spans everything from picking/researching a payload
+to designing the integration of that payload to the CubeSat, with all the features required by the CSLI\@. The primary objective of this phase is to create a
```
For payload integration, to what level is the rest of the cubesat developed? Is each major subsystem designed? Will subsystems be purchased Commercial Off The Shelf (COTS) or developed in-house? What features are required by the CSLI? How is integration defined? Does that include the mechanical mounting of payload hardware within the cubesat? Are wiring and power diagrams required? Will thermal management analysis of the whole cubesat or just the payload subsystem be performed?

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+  % This is the main purpose or main goal the project hopes to achieve.
+This part of the project, Phase I, will concern itself with the design and research of the CubeSat. This spans everything from picking/researching a payload
+to designing the integration of that payload to the CubeSat, with all the features required by the CSLI\@. The primary objective of this phase is to create a
+fully developed CAD model, which includes a fully researched and developed scientific payload, and the proper documentation concerned with the electronic features such that the team can write a fully detailed report on
```
What defines a fully researched and developed scientific payload? Will the physical payload hardware be developed and tested? 

What documentation of electronic features is expected? Is that wiring diagrams or board layouts? Does that concern power distribution or data handling? Does it cover software? Is the electronic work scoped to just the electronics for the payload or also the whole cubesat?

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+% In this section, explain how the project would benefit SPEX members as students,
+% space enthusiasts, and young professionals.
+This project would be beneficial to SPEX for the following reasons; experience (project management, engineering, coding, etc.) given to current members, opportunities
+for data analytics on collected data, PR, provides base for further funding and sponserships.
```
Can you describe what data will be collected? Will there be physical demonstrations or prototypes or will data be generated via simulations? What data processing do you expect to do? What variables are being measured and collected?

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+% space enthusiasts, and young professionals.
+This project would be beneficial to SPEX for the following reasons; experience (project management, engineering, coding, etc.) given to current members, opportunities
+for data analytics on collected data, PR, provides base for further funding and sponserships.
+The PR for this would be great for getting recognition in the aerospace community. A successful CubeSat is
```
I am still a bit confused. Does this project result in a finished cubesat? The wording here implies the PR benefits come with a successful cubesat flight? What are the immediate PR benefits of the work described in this project?

runphilrun replied:
+1 to above questions

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+The PR for this would be great for getting recognition in the aerospace community. A successful CubeSat is
+difficult to attain. If SPEX is able to excute this project well, it would be a serious step in being national recognized as an aerospace/astronautical research group.
+The marketing team at RIT SPEX would be able to leverage this project to get interested companies to donate supplies, money, or their time so RIT SPEX can build
+and grow in their research. The experience for the team would be great because it would bring a project from pure design and research to flight. Allowing students
```
Again, it seems like this describes actual construction of a flight ready cubesat rather than scientific payload exploration studies.

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+  % What path do you anticipate the project to take?
+This project will start with a small design and research team. That team will be responsible for picking and appropriate payload. Once the payload is chosen, the
+design of the CubeSat will begin. The team will bring in members that have experience in the relevant categories, such as radio communication, structural engineering,
+software and firmware development, etc. The team will be responsible for creating thorough documentation on every aspect of the CubeSat to ensure future members will be able to pick it up and
```
Will firmware be written for this project? Will they use development board or other hardware?

venku122 replied:
What kind of documentation is intended here? Are these assembly/manufacturing instructions or just notes on decisions and other research activities?

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+\section{Implementation}
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+This project will start with a small design and research team. That team will be responsible for picking and appropriate payload. Once the payload is chosen, the
```
>~and~ an

runphilrun commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+\makenomenclature{}
+
+% set title. choose something as descriptive and precise as possible. Descriptive > sounding cool. remember this!
+\title{CubeSat Launch Initiative Phase I\@: \newline Design \& Research}
```
use `\\` instead of `\newline`

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+    \toprule
+    Step & Description \\
+    \midrule
+    1 & Define mission needs \\
```
Nice to see these steps listed out. 

It would be nice to see more detail on how these steps apply to the project, and what will actually be done during each step. Tangible deliverables can help the team stay focused.

runphilrun commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+
+\subsection{Long-Term Vision}
+\label{sec:vision}
+The long-term vision of this project is to launch a CubeSat. This project begins with the most important phase, the initial design and research. Once this is complete the team will move onto
```
You say that this is Phase 1, but you don't define Phases 2+. Define the future phases in this section leading up to a flight-ready cubesat ready for orbit.

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+\label{subsec:deliverables}
+  % When all is said and done, what will you have to show for it?
+  % Examples: Hardware, software, poster, ImagineRIT demo, presentations, technical papers...
+The team will make 2 or 3 postings on the RIT SPEX website, per semester, with progress and science updates to keep the project in outside contributors/persons of
```
Nice 👍

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+interest's field of view. The team will be required to have a poster for Imagine RIT and the undergraduate research symposium in the spring and summer terms, respectively.
+
+The documentation for a project of this size and length is essential. Since the project is going to take a few semesters, at minimum, to complete. It is important
+to have enough documentation on each aspect to be able to bring new members up-to-speed as easy and quick as possible. The documentation should cover the following; fully end-to-end CubeSat assembly,
```
What defines enough? What specific topics will be covered and in what way? Is it meeting notes and powerpoints or official reports that cover topics in-depth?

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+their progress and contribute to a master logbook in which documentation will be recorded for future reference. The documents can range from code to articles on specific calculations
+or designs, etc. Powerpoints are preferred for presenting to other members.
+
+This phase of the project will be considered complete when the following is delivered; a fully developed CAD model, electrical diagrams describing the electrical functionality,
```
What specific electrical diagrams/PCBs are planned to be designed? Does it include the science board or all PCBs within the cubesat?

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+The milestones for this project include surpassing each step laid out in \autoref{sec:implementation}. The more general project milestones are choosing a payload,
+designing the payload, CubeSat structure, CubeSat avionics and software. Then also creating a risk assessment based on the first design iteration.
+
+Once these first milestones are completed the team will go through a concept design review. This will give them early feedback based on their design decisions and
```
👍

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+  % Be as detailed as you can, but it's okay if there are unknowns.
+  % At the very least, specify how many semester you expect the project to take until it reaches completion.
+The milestones for this project include surpassing each step laid out in \autoref{sec:implementation}. The more general project milestones are choosing a payload,
+designing the payload, CubeSat structure, CubeSat avionics and software. Then also creating a risk assessment based on the first design iteration.
```
Does this cover designing the software or writing software?

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+Once these first milestones are completed the team will go through a concept design review. This will give them early feedback based on their design decisions and
+allow them the opportunity to address concerns before the system design is frozen. Then the team will iterate upon their design further, figure out more details and
+go back for a project design review with another risk assessment. After the project design review the team will make final design iterations and prepare for the funding phase.
+Preparation for funding will include making many documents and graphics to show the project to potential investors and or to please already dedicated investors.
```
What specific documents do you plan to produce? Is this another sponsorship packet, resources for a kickstarter/crowdfunding campaign, or something else? What potential investors are considered? Is this a commercial product?

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+that may extend beyond the reach of their speciality. While it extends beyond their speciality, it will not extend beyond their capability. In the event that is unavoidable, outside
+help will be sought through professors or the appropriate student body.
+
+In general, it is expected that mechanical engineers will be profficient in Solidworks and MATLAB. Solidworks is the chosen 3D modeling software due wide availability and experience.
```
>~due~ due to

runphilrun reviewed (commented):
will read later

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+help will be sought through professors or the appropriate student body.
+
+In general, it is expected that mechanical engineers will be profficient in Solidworks and MATLAB. Solidworks is the chosen 3D modeling software due wide availability and experience.
+MATLAB will be used for calculating orbits or performing mechanical analysis. It is not required for people that have a refined coding ability. Mechanical engineers will need to
```
Orbit calculations would be very interesting. It would be nice to see that defined as an objective or deliverable.

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+
+\subsection{Funding Requirements}
+  % Estimate costs that would be needed to meet objectives.
+The funding requirements for this design phase are minimal. There may be some cost during feasability studies if a particular technology must be demonstrated.
```
What specific technology demonstrators would be built?

jprks replied:
Unknown at this time. It completely depends on the selected payload.

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+\subsection{Faculty Support}
+  % Identify faculty that will be involved (or would need to be involved) to meet objectives.
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+Faculty support is difficult to gauge based on the ambiguity of the payload. Since the payload is completely unknown at this time, the extent of faculty support will be considered when
```
Will faculty support be sought out during the payload exploration phase?

jprks replied:
No, we will find them when we choose the payload.

venku122 commented on CSLI/CSLI.tex:
```diff
@@ -0,0 +1,352 @@
+The team that works on this project does not need to be on the future stages of the project. The goal of the rigorous documentation is to allow any and all members
+to pick up where this team left off, with minimal reading to get caught up to speed.
+\section*{Acknowledgements}
+The author would like to thank Dr.~Bill Destler and Rebecca Johnson for being exemplary humans, Anthony Hennig for founding RIT Space Exploration, and all the SPEX members that continue to invest their time and energy into the pursuit of space exploration.
```
This section should be customized to be directly applicable or removed.

venku122 reviewed (changes requested):
CSLI/SPEXpdd.pdf
CSLI/figs/project-life-cycle.png
CSLI/figs/project life cycle.vsdx
CSLI/SPEXpdd.pdf

Can be removed.

Overall, I think the project definition is too vague. Activities should be clarified and well-defined, so successful completion of those activities can be measured.

runphilrun reviewed (commented):
I read through previous comments from other users and agree with pretty much all of them. It's not clear to me if the deliverable is intended to be a white paper, a paper entry to a competition, or flight-ready hardware? What will the cubesat do?

Commit:
Updated from comments

Made the updates from the comments. Hopefully this carlifies the project intentions.

Commit:
Addressed more comments

Made changes that were suggested by @venku122 and @runphilrun.

venku122 commented:
@jprks No PDD needs to be deleted if its not immediately approved. The PR can be merged and its a great starting point for future projects.
